securing_apps/topics/oidc/java/params_forwarding.adoc

```diff
@@ -19,7 +19,11 @@ The supported parameters are:
 
 * scope
 
-* prompt
+* prompt - Values supported by {project_name} are currently:
+** `login` - SSO will be ignored and {project_name} login page will be always shown even if user is already authenticated
+** `none` - Login page will never be shown, but user will be redirected back to the application with the error in case that he
+is not yet authenticated. This allows you for example to create filter/interceptor on the application side and show some custom error page to the user.
+See more details in the specification.
 
```

Comment:
* prompt - {project_name} supports these settings:
** `login` - SSO will be ignored and the {project_name} login page will be always shown, even if the user is already authenticated
** `none` - The login page will never be shown; instead the user will be redirected to the application, with an error if the user is not yet authenticated. This setting allows you to create a filter/interceptor on the application side and show a custom error page to the user.
See more details in the specification.